recipes-core/initscripts/files/mdev/ttyUSB.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Look for ttyUSB devices which match the known location of the USB to UART
+# adapter connected to the Cypress Bluetooth on the mangOH Yellow. When such a
+# device is identified, either spawn or kill btattach depending on whether the
+# UART is being added or removed respectively.
+if [ $DEVNAME ] && [ $DEVPATH == "/devices/7c00000.hsic_host/usb1/1-1/1-1.2/1-1.2:1.0/$DEVNAME/tty/$DEVNAME" ]
+then
+    PIDFILE="/var/run/btattach.$DEVNAME.pid"
+    if [ $ACTION == 'add' ]
+    then
+        btattach -P h4 -B /dev/$DEVNAME &
+        echo -n $! > $PIDFILE
+    elif [ $ACTION == 'remove' ]
+    then
+        if [ -f $PIDFILE ]
+        then
+            kill `cat $PIDFILE`
```

Comment:
Check that is was kill'ed  before you remove the pidfile. Otherwise, consider "-9" it.